pkg/ccr/base/spec.go

```diff
@@ -297,6 +297,67 @@ func (s *Spec) GetAllTables() ([]string, error) {
 	return tables, nil
 }
 
+func (s *Spec) queryResult(querySQL string, queryColumn string, errMsg string) ([]string, error) {
+	db, err := s.ConnectDB()
+	if err != nil {
+		return nil, err
+	}
+
+	rows, err := db.Query(querySQL)
+	if err != nil {
+		return nil, xerror.Wrap(err, xerror.Normal, querySQL+" failed")
+	}
+	defer rows.Close()
+
+	var results []string
+	for rows.Next() {
+		rowParser := utils.NewRowParser()
+		if err := rowParser.Parse(rows); err != nil {
+			return nil, xerror.Wrap(err, xerror.Normal, errMsg)
+		}
+		result, err := rowParser.GetString(queryColumn)
+		if err != nil {
+			return nil, xerror.Wrap(err, xerror.Normal, errMsg)
+		}
+		results = append(results, result)
+	}
+
+	return results, nil
+}
+
+func (s *Spec) GetAllViewsFromTable(tableName string) ([]string, error) {
+	log.Debugf("get all view from table %s", tableName)
+
+	var results []string
+	// first, query information_schema.tables with table_schema and table_type, get all views' name
+	querySql := fmt.Sprintf("SELECT table_name FROM information_schema.tables WHERE table_schema = '%s' AND  table_type = 'VIEW'", s.Database)
```

Comment:
`desc ${tableName} all` might be easier and more efficient than this.

See https://doris.apache.org/docs/1.2/query-acceleration/materialized-view/#query-materialized-views.

Reply:
'desc ${tableName} all' don't support common view, materialized view is supported